tests/reg_issue662_match_cases_missing/issue662.fz.expected_err

```diff
@@ -7,4 +7,10 @@ Match expression at --CURDIR--/issue662.fz:30:3:
 --^
 To solve this, add a case.  If a case exists, check that the indentation is deeper than that of the surrounding 'match' expression
 
-one error.
+
+--CURDIR--/issue662.fz:26:1: error 2: Failed to infer result type for feature 'issue662'.
+issue662 =>
+^^^^^^^^
+To solve this, please specify a result type explicitly.
+
+2 errors.
```

Comment:
This is a subsequent error that should be suppressed. Would be good to do this within this PR.

Reply:
@fridis Not sure what is the best approach here?

Reply:
I think this could be solved by setting `_type` to `Types.t_ERROR` at the end of `Match.resolveTypes()` as follows:
```
        if (!missingMatches.isEmpty() && ok)
          {
            AstErrors.missingMatches(pos(), cgs, missingMatches);
            _type = Types.t_ERROR; 
          }
```